src/index.js

```diff
@@ -188,9 +190,53 @@ class IOpipeWrapperClass {
       this.originalContext.done(err, data);
     });
   }
+  metric(keyInput, valueInput) {
+    const key = String(keyInput);
+    var numberValue = undefined;
+    var stringValue = undefined;
+    if (key.length >= 256) {
+      console.warn(
+        `Metric with key name ${key} is longer than allowed length of 256, metric will not be saved`
+      );
+      return;
+    }
+    if (valueInput === undefined) {
+      console.warn(
```

Comment:
A user's `value` may be undefined and help them to debug an issue, JSON.stringify can help us here

Reply:
True, that'd be the same behavior as current `.log`